scripts/utils.sh

```diff
@@ -8,25 +10,36 @@ function url_reachable() {
     return $?
 }
 
-
 function spawn_port_forwarding_command() {
-  kill_portforwarding_loop $1 $2 $3
-  run_in_background "scripts/port_forwarding_loop.sh $1 $2 $3"
+  service_name=$1
+  external_port=$2
+
+  cat << EOF > /etc/xinetd.d/${service_name}
+service ${service_name}
+{
+  flags		= IPv4
+  bind		= $(hostname -I | cut -d' ' -f1)
+  type		= UNLISTED
+  socket_type	= stream
+  protocol	= tcp
+  user		= root
+  wait		= no
+  redirect	= $(minikube ip) $(kubectl describe svc/${service_name} -n assisted-installer | grep NodePort | awk '{print $3}' | sed 's/\/.*//')
```

Comment:
do u have the kubeconfig there? I mean no export is required?

Reply:
yes it is. u need this:

 --kubeconfig=${KUBECONFIG}

Reply:
U can use this:
kubectl get svc/bm-inventory -n assisted-installer  -o=jsonpath='{.spec.ports[0].nodePort}'

More clear in my opinion.

Reply:
I agree.